Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -118,6 +118,26 @@ python::tuple calcCrippenDescriptors(const RDKit::ROMol &mol,
 
 #ifdef RDK_BUILD_DESCRIPTORS3D
 
+#ifdef RDK_HAS_EIGEN3
+python::tuple calcANIAtomicEnvironmentVector(const RDKit::ROMol &mol,
+                                             int confId) {
+  std::vector<std::vector<double>> results;
+  auto aev = RDKit::Descriptors::ANI::AtomicEnvironmentVector(mol, confId);
+  for (auto i = 0; i < aev.rows(); i++) {
+    std::vector<double> row;
+    for (auto j = 0; j < aev.cols(); j++) {
+      row.push_back(aev(i, j));
+    }
+    results.push_back(row);
+  }
+  python::list result;
+  for (auto &res : results) {
+    result.append(res);
+  }
```

Comment:
You could simplify this and save a bunch of copying by replacing it with:
```suggestion
  python::list result;
  for (auto i = 0; i < aev.rows(); i++) {
    python::list row;
    for (auto j = 0; j < aev.cols(); j++) {
      row.append(aev(i, j));
    }
    result.append(python::tuple(row));
  }
```